### PR TITLE
feat(eng-863): implement test and status functionality for all integrations

### PR DESCRIPTION
**Pull request by pashafateev**

### CODE REVIEW

File: integrations/twilio/webhooks/auth.go [hunk 1]
```diff
@@ -52,10 +57,50 @@ func (h handler) HandleAuth(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// TODO(ENG-1156): Test the authentication details.
+	api := TwilioTest{Vars: Vars{
+		AccountSID: accountSID,
+		Username:   username,
+		Password:   password,
+	}}
+	ctx := context.Background()
```
Comment (itayd) [inline integrations/twilio/webhooks/auth.go:65]: use `r.Context()`

File: integrations/twilio/webhooks/auth.go [hunk 2]
```diff
@@ -52,10 +57,50 @@ func (h handler) HandleAuth(w http.ResponseWriter, r *http.Request) {
...
 }
+
+type TestResponse struct {
+	AccountSID   string `json:"AccountSID"`
+	FriendlyName string `json:"FriendlyName"`
+	Status       string `json:"Status"`
+}
+
+func (a TwilioTest) Test(ctx context.Context) (*TestResponse, error) {
+	// Create a new Twilio client using the provided username and password
```
Comment (itayd) [inline integrations/twilio/webhooks/auth.go:88]: Just inline this logic above, no need for another struct.

File: integrations/chatgpt/client.go [hunk 3]
```diff
@@ -40,6 +42,22 @@ func New(vars sdkservices.Vars) sdkservices.Integration {
...
+		if !cid.IsValid() {
+			return sdktypes.NewStatus(sdktypes.StatusCodeWarning, "init required"), nil
+		}
+
+		_, err := cvars.Get(ctx, sdktypes.NewVarScopeID(cid))
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		return sdktypes.NewStatus(sdktypes.StatusCodeOK, "using api key"), nil
```
Comment (itayd) [inline integrations/chatgpt/client.go:61]: "initialized"

**Comment by pashafateev** [comment]
As a result of #583 I decided to split this PR into two to separate test and status functionality

**Pull request closed without merging**